TS03-Subset-Data-and-No-Data-Values.Rmd

```diff
@@ -72,18 +70,16 @@ refresher on setting the working directory, please view the <a href="XXX" target
 
 </div>
 
-In [{{site.baseurl}} / LINKHERE](Lesson 02 - ) we imported a
-`.csv` time series dataset and learned out to convert data fields into date-time
-classes.  We'll use that micrometereology data (from file 
+In [{{site.baseurl}} / LINKHERE](Lesson 01 - ) we imported a
+`.csv` time series dataset and in [{{site.baseurl}} / LINKHERE](Lesson 02 - ) we 
+learned out to convert data fields into date-time classes.  We'll use that data (from file 
 `hf001-10-15min-m.csv`) to explore the relationship between our variables of 
-interest, air temperature, precipitation, and photosynetheically active 
-radiation (PAR) in regards to plant phenology.  
+interest, air temperature, precipitation, and PAR in regards to plant phenology.  
```

Comment:
@mguarinello Can you add  "photosynthetically active radiation" back in prior to PAR.  We need to state it once in each lesson as individuals may not be going through all lessons in order.